ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/PaywallDialog.kt

```diff
@@ -38,7 +38,10 @@ fun PaywallDialog(
         }
     }
     if (shouldDisplayDialog) {
-        val dismissRequest = { shouldDisplayDialog = false }
+        val dismissRequest = {
+            paywallDialogOptions.dismissRequest?.invoke()
+            shouldDisplayDialog = false
+        }
 
         Dialog(
             onDismissRequest = dismissRequest,
```

Comment:
It makes sense to pass this here, but this is also it makes sense to pass this here, but this is modifying line 50 too